specifyweb/frontend/js_src/lib/components/TreeView/Row.tsx

```diff
@@ -168,16 +194,28 @@ export function TreeRow({
                     ? 'outline outline-1 outline-blue-500'
                     : ''
                 }
+                ${
+                  isSecondFocused === row.nodeId
+                    ? 'outline outline-1 outline-green-500'
+                    : ''
+                }
               `}
               forwardRef={isFocused ? handleRef : undefined}
               key={rankId}
               style={{
                 color:
                   typeof row.acceptedId === 'number' ? synonymColor : undefined,
               }}
-              onClick={({ metaKey, shiftKey }): void =>
-                metaKey || shiftKey ? handleFocusNode([]) : handleToggle(false)
-              }
+              onClick={({ metaKey, shiftKey }): void => {
+                metaKey || shiftKey ? handleFocusNode([]) : handleToggle(false);
+                isFirst
```

Comment:
1. Rather than do this on every node (and have to pass down the setLastTreeClicked prop, why not just set the event listener in Tree or TreeView component?
2. Instead of "click", you should probably be listening for focus - click is mouse only, where as focus is both mouse and keyboard
3. there lastTreeClicked shouldn't really ever be undefined - just set it to `"first"` by default - will simplify things a bit as no need to check for undefined anymore